lib/surface/directive/debug.ex

```diff
@@ -1,32 +1,31 @@
 defmodule Surface.Directive.Debug do
-  use Surface.Directive
-
-  def extract({":debug", {:attribute_expr, value, expr_meta}, attr_meta}, meta) do
-    %AST.Directive{
-      module: __MODULE__,
-      name: :debug,
-      value: directive_value(value, Map.merge(meta, expr_meta)),
-      meta: Helpers.to_meta(attr_meta, meta)
-    }
+  use Surface.Directive,
+    extract: [
+      name: ":debug",
+      type: :static_list
+    ]
+
+  def handle_value(%AST.Text{value: true}, meta) do
```

Comment:
I think we should rename `AST.Text` to `AST.Literal` in the near future. After all, it can hold not only text but also boolean and integer.